sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1089,6 +1089,22 @@ def _lambda_handler(blocked_resources: Set['resources_lib.Resources'],
             FailoverCloudErrorHandlerV2._default_handler(
                 blocked_resources, launchable_resources, region, zones, error)
 
+    @staticmethod
+    def _aws_handler(blocked_resources: Set['resources_lib.Resources'],
+                     launchable_resources: 'resources_lib.Resources',
+                     region: 'clouds.Region',
+                     zones: Optional[List['clouds.Zone']],
+                     error: Exception) -> None:
+        output = str(error)
+        logger.info(f'AWS handler error: {output}')
```

Comment:
Minor change:
```suggestion
        logger.info(f'AWS handler error: {error}')
```
Since we only use this variable once, I think this is cleaner.

Reply:
OK, updated